bcgov/Unity
# Dev To Main Release 2023-12-19

QA Testable:

[AB#12551](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/12551) - fix assessment summary date format
[AB#10661](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/10661) - create a split in databases (Manager/host and default tenant database)
[AB#9262](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9262) - application submissions by sector pie chart in dashboard
[AB#9859](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9859) - forms versions management and fixes 
[AB#9931](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9931) - configurable columns and layout in application list
[AB#10699](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/10699) - increase length of certain application fields (this was already deployed as a hotfix previously)
[AB#8389](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/8389) - project info tab
[AB#8910](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/8910) - make decline rationale required if declined
[AB#9257](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9257) - application status overview pie chart in dashboard
[AB#10547](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/10547) - export application list fields (check)
[AB#9260](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9260) - application submission economic reagion breakdown dashboard
[AB#10633](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/10633) - fix unknown component: orgbook

Non QA Testable:

[AB#12598](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/12598) - add electoral district map and workflow ignore metabase folder
[AB#12574](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/12574) - add tenant db connection string to openshift config
[AB#12761](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/12761) - update ABP cli command to install V7.4.4

Partial - still WIP:

[AB#9229](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9229) - application project info tab and fields
